app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -515,10 +521,8 @@ Mautic.sanitizeHtmlBeforeSave = function(htmlContent) {
     // Remove the slot focus highlight
     htmlContent.find('[data-slot-focus], [data-section-focus]').remove();
 
-    customHtml = Mautic.domToString(htmlContent);
```

Comment:
This was added by @escopecz for the Apply button function. Will removing this cause issues?

Reply:
I restored the line